### ISSUE TITLE
Something went wrong while executing command "//paste"

### ISSUE THREAD

**Issue by koenpro** [cuberite/WorldEdit#127]
When I type the command //cut or //paste I get an error message saying:

Something went wrong while executing command "//paste"

```
[13:24:32] Write: Requesting datatypes not present in the cBlockArea. Got only 0x03, requested 0x23
[13:24:32] Parameters on the stack
[13:24:32] LUA: Write: Requesting datatypes not present in the cBlockArea. Got only 0x03, requested 0x23
[13:24:32] Stack trace:
[13:24:32]   [C](-1): Write
[13:24:32]   Plugins/worldedit/Classes/Clipboard.lua(146): Paste
[13:24:32]   Plugins/worldedit/Commands/Clipboard.lua(103): (no name)
[13:24:32] Stack trace end
[13:24:32] Error in <attached> calling function <callback>()
```

**Comment by Seadragon91** [comment]
There is a open pull request that should fix the problem #126

**Comment by Seadragon91** [comment]
Should be now fixed.
